src/Stripe.net/Infrastructure/Middleware/ParserPlugin/AdditionalOwnerPlugin.cs

```diff
@@ -12,8 +12,15 @@ public bool Parse(ref string requestString, JsonPropertyAttribute attribute, Pro
             if (attribute.PropertyName != "legal_entity[additional_owners]") return false;
 
             var owners = ((List<StripeAccountAdditionalOwner>) property.GetValue(propertyParent, null));
-            if (owners == null) return true;
 
```

Comment:
instead of this (or maybe in addition?), why not just check for null, apply the parameter and return true? ...

I realize this would change the test a bit. You would just set it to null instead of a new empty list. 

Alternatively, your check could just be owners.Count == 0 || owners != null. I think this would allow both ways to work.